monai/transforms/intensity/array.py

```diff
@@ -1423,15 +1419,15 @@ class KSpaceSpikeNoise(Transform, Fourier):
         with `log-intensity = 14`.
     """
 
+    backend = [TransformBackends.TORCH, TransformBackends.NUMPY]
+
     def __init__(
         self,
         loc: Union[Tuple, Sequence[Tuple]],
         k_intensity: Optional[Union[Sequence[float], float]] = None,
-        as_tensor_output: bool = True,
```

Comment:
I think currently what we do is to keep it and have a deprecated warning https://github.com/Project-MONAI/MONAI/blob/790fc8f572322c6a4c1ab527acfb27447be4bb66/monai/transforms/spatial/array.py#L1116

Reply:
Thanks!